test/release/examples/primers/modules.chpl

```diff
@@ -8,60 +8,65 @@
     * namespace control within a module
     * external access to a module's symbols
     * namespace control when using a module, including:
+
       * unlimited
       * explicit exclusion of symbols
       * explicit inclusion of symbols
       * renaming included symbols
+
     * cross-file module access
 */
 
 
 /* A module is a grouping of code - each program consists of at least one
    module, and every symbol is associated with some module.  If all the code of
-   a file is not enclosed in an explicit module, defined using the 'module'
+   a file is not enclosed in an explicit module, defined using the ``module``
    keyword, then the file itself is treated as a module with the same name as
    the file (minus the .chpl suffix).  The compiler can be directed to include
    modules in separate files by naming them on the command line, utilizing the
-   -M flag, etc. (see the man page for exact details).
+   ``-M`` flag, etc. (see the :ref:`man-chpl` page for exact details).
```

Comment:
Right now, :ref:`man-chpl` renders to just `chpl`. Does something like this fix this?

    (see the :ref:`man page <man-chpl>` for exact details)